java/ql/src/Violations of Best Practice/Dead Code/UnusedLocal.ql

```diff
@@ -29,6 +29,4 @@ where
   // Rules about catch clauses belong in an exception handling query
   not exceptionVariable(ve) and
   not enhancedForVariable(ve)
-select v,
-  "Unused local variable " + v.getName() +
-    ". The variable is never read or written to and should be removed."
+select v, "The value assigned to local variable '" + v.getName() + "' is never used."
```

Comment:
The new message says that the variable is being assigned a value, but the old message specifically says that it wan't written to. Which is it?

Reply:
It seems to be "wasn't written to". The query behaves slightly different compared to the other languages.  

I'll revert this change.